src/sagas.js

```diff
@@ -664,31 +669,54 @@ function* fetchAllNetworks () {
   }
 }
 
-function* schedulerWithFixedDelay (action) {
-  logDebug('Starting schedulerWithFixedDelay() scheduler')
+function* delayedRemoveActiveRequest ({ payload: requestId }) {
+  yield delay(500)
+  yield put(removeActiveRequest(requestId))
+}
 
-  // TODO: do we need to stop the loop? Consider takeLatest in the rootSaga 'restarts' the loop if needed
-  while (true) {
-    yield delay(AppConfiguration.schedulerFixedDelayInSeconds * 1000)
-    logDebug(`schedulerWithFixedDelay() event after delay of ${AppConfiguration.schedulerFixedDelayInSeconds} seconds`)
+function* startSchedulerWithFixedDelay (action) {
+  // if a scheduler is already running, stop it
+  if (Selectors.isSchedulerEnabled()) {
+    yield put(enableScheduler({ enabled: false }))
```

Comment:
Why do you have here `yield put(enableScheduler({ enabled: false }))` if `stopSchedulerFixedDelay()` doing the same thing?

Reply:
Ah - that is confusing.  I'll streamline that a bit.

The `start` can call `stop` directly, and the `stop` can use a different action string so it doesn't share with the `take` in the `race` in `schedulerWithFixedDelay`